wadas/domain/database.py

```diff
@@ -209,9 +213,22 @@ def create_database(self):
         """Method to create database by creating all tables"""
 
         if DataBase.wadas_db_engine:
-            Base.metadata.create_all(DataBase.wadas_db_engine)
-            return True
+            try:
+                Base.metadata.create_all(DataBase.wadas_db_engine)
+            except SQLiteOperationalError:
+                logger.exception("OperationalError during database creation.")
+                return False
+            except SQLAlchemyError:
+                logger.exception("SQLAlchemyError during database creation:.")
+                return False
+            except Exception:
+                logger.exception("Unexpected error during database creation.")
+                return False
```

Comment:
You can remove all return False from here, and just add a single one after the else: block (so, just after the all try/except/else statement).